cluster/gce/upgrade.sh

```diff
@@ -179,8 +179,8 @@ function upgrade-nodes() {
 function setup-base-image() {
   if [[ "${env_os_distro}" == "false" ]]; then
     echo "== Ensuring that new Node base OS image matched the existing Node base OS image"
-    node_os_distribution=$(get-node-os "${NODE_NAMES[0]}")
```

Comment:
this was meant to be a local variable. I did not realize that it is actually necessary. Err......